js/modules/k6/metrics/metrics.go

```diff
@@ -37,7 +37,7 @@ import (
 
 type Metric struct {
 	metric *stats.Metric
-	core   modules.InstanceCore
+	core   modules.InstanceParams
```

Comment:
Just an idea.

What about `vu` as a field name instead of `core`?

So it'll be natural to type:
```golang
state := m.vu.State()
```

Instead of:

```golang
state := m.core.GetState()
```